src/main/java/dynamok/source/ConnectorConfig.java

```diff
@@ -43,6 +46,12 @@
                     throw new ConfigException("Invalid AWS region: " + regionName);
                 }
             }, ConfigDef.Importance.HIGH, "AWS region for the source DynamoDB.")
+            .define(Keys.ACCESS_KEY_ID, ConfigDef.Type.PASSWORD, "",
+                    ConfigDef.Importance.LOW, "Explicit AWS Access Credentials" +
```

Comment:
Need a fullstop and space at the end of the first sentence so it formats correctly when concatenated.